File: benchmark/results/java-invocation-script/run-005/a-mcp/output/inventory/value_objects/Reservation.java
package com.metaengine.demo.inventory.value_objects;

import java.time.OffsetDateTime;

// Reservation value object.

public class Reservation {
    public String id;
    public OffsetDateTime createdAt;
    public OffsetDateTime updatedAt;
    public String name;
    public String description;

    public Reservation(String id, OffsetDateTime createdAt, OffsetDateTime updatedAt, String name, String description) {
        this.id = id;
        this.createdAt = createdAt;
        this.updatedAt = updatedAt;
        this.name = name;
        this.description = description;
    }
}

File: benchmark/results/python-multilang/run-002/b-baseline/output/metaengine_demo/inventory/value_objects/reservation.py
"""Reservation value object for the inventory domain."""

from dataclasses import dataclass
from datetime import datetime


@dataclass(frozen=True)
class Reservation:
    """Immutable reservation of stock against an order."""

    id: str
    createdAt: datetime
    updatedAt: datetime
    name: str
    description: str
